gcalcli/gcalcli.py

```diff
@@ -1632,7 +1648,7 @@ def main():
         else:
             tmp_argv = argv
 
-        (parsed_args, junk) = parser.parse_known_args(tmp_argv)
+        (parsed_args, unparsed) = parser.parse_known_args(tmp_argv)
```

Comment:
This could be in a separate PR because it's a bugfix. Users on `-git` will have to wait days for this fix.

Reply:
That was already fixed in my PR I think.

Reply:
@laser - Ah, I see what you're saying. This is a current bug in master, you're right. @jcrowgey - I'll do another PR isolating this fix

Reply:
Fixed in PR #413.